pydash/pydash_mail/__init__.py

```diff
@@ -0,0 +1,3 @@
+from flask_mail import Mail
```

Comment:
Why turn this into a separate package? As it stands it's just a very (very) thin wrapper around Flask-Mail`

Reply:
I thought this was the most appropriate place to put the overarching Mail() object, such that it can be used in different places. (It functions like the controller object that interacts with the smtp server)

Reply:
I was mostly wondering since we're not using lazy initialization for other Flask plugins - ie `mail.init_app(flask_webapp)` instead of `mail = Mail(flask_webapp)`. Although your way makes sense, and doesn't lead to a tight coupling between `pydash_web` and `pydash_mail`. :+1:

Reply:
The issue with initialising the object in `pydash_web.__init__.py` would be that it'd create circular dependencies: `pydash_web` importing the `verification` module, which in turn imports `pydash_web` to get access to that single `Mail()` object.

Reply:
Exactly. This is fine :smile: